### ISSUE TITLE
It is recommended to include a space before the trailing “/>” in empty XML elements

### ISSUE THREAD

**Issue by greg-raven** [gridsome/gridsome#122]
## Description

Meta tags are missing a space before the closing />

### Steps to reproduce

Run generate build and attempt to validate the resulting code.

### Expected result

<meta data-vue-tag="true" charset="utf-8" />

### Actual result

<meta data-vue-tag="true" charset="utf-8"/>

**Comment by zacnomore** [comment]
Source: https://www.w3.org/TR/xhtml1/#guidelines
Section C2

**Comment by zacnomore** [comment]
Interestingly, the xhtml spec requires the trailing `/` wheras the HTML5 spec states that it's optional.
https://html.spec.whatwg.org/multipage/syntax.html#start-tags

> Then, if the element is one of the void elements, or if the element is a foreign element, then there may be a single U+002F SOLIDUS character (/). This character has no effect on void elements, but on foreign elements it marks the start tag as self-closing.

Using the [w3c validation service](https://validator.w3.org/), it validates the meta tags as-is. I tested this on my own site, you can see that testing at [the w3c validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fzacharysvoboda.com%2F).

I believe that would mean the same for the trailing space as well. I suggest that we close this ticket, it does not seem to have a positive impact to make the change and the W3C seems to support the current usage.

@greg-raven do you have an opinion in light of this information?

**Comment by greg-raven** [comment]
Yes. My validator is fine with either no / character or a space and then the / character, but not with the / character by itself.

**Comment by zacnomore** [comment]
What validator are you using? Seems like this is already falling in spec
